pom.xml

```diff
@@ -63,6 +63,8 @@
         <licenses.version>6.0.0-SNAPSHOT</licenses.version>
         <checkstyle.suppressions.location>checkstyle/suppressions.xml</checkstyle.suppressions.location>
         <hadoop.version>2.7.7</hadoop.version>
+        <kafka.scala.version>2.12</kafka.scala.version>
```

Comment:
Are these taken from the other PR related to ITs?

Reply:
yes